model/registry/dns.yaml

```diff
@@ -0,0 +1,20 @@
+groups:
+  - id: registry.dns
+    type: attribute_group
+    prefix: dns
+    brief: >
+      This document defines the shared attributes used to report a DNS query.
+    attributes:
+      - id: question.name
+        type: string
+        stability: experimental
+        brief: The name being queried.
+        examples: ["www.example.com", "opentelemetry.io"]
+        note: >
+          The name being queried.
```

Comment:
I think note shouldn't repeat the `brief` content

Reply:
great catch!